minet/facebook/mobile_scraper.py

```diff
@@ -60,6 +66,26 @@ def resolve_relative_url(url):
     return urljoin(FACEBOOK_MOBILE_URL, url)
 
 
+def clean_likes(text):
```

Comment:
Move this to utils and use it in `cli.facebook.url_likes` since it comes from it.